README.md

```diff
@@ -125,15 +125,19 @@ Your config folder will be mounted in `/tmp/docker-mailserver/`. To understand h
 `restart: always` ensures that the mail server container (and Filebeat/ELK containers when using the mail server together with ELK stack) is automatically restarted by Docker in cases like a Docker service or host restart or container exit.
 
 #### Exposed ports
-* 25 receiving email from other mailservers
-* 465 SSL Client email submission
-* 587 TLS Client email submission
-* 143 StartTLS IMAP client
-* 993 TLS/SSL IMAP client
-* 110 POP3 client
-* 995 TLS/SSL POP3 client
-
-`Note: Port 25 is only for receiving email from other mailservers and not for submitting email. You need to use port 465 or 587 for this.`
+
+| Protocol | Opt-in Encryption<sup>1</sup> | Enforced Encryption | Purpose              |
+|----------|-------------------------------|---------------------|----------------------|
+| SMTP     | 25                            | N/A                 | Transfer<sup>2</sup> |
+| ESMTP    | 587                           | 465<sup>3</sup>     | Submission           |
+| POP3     | 110                           | 995                 | Retrieval            |
+| IMAP4    | 143                           | 993                 | Retrieval            |
+
+1. A connection *may* be secured over TLS when both ends support `STARTTLS`. On ports 110, 143 and 587, `docker-mailserver` will reject a connection that cannot be secured. Port 25 is [required](https://serverfault.com/questions/623692/is-it-still-wrong-to-require-starttls-on-incoming-smtp-messages) to support insecure connections.
+2. Receives email and filters for spam and viruses. For submitting outgoing mail you should prefer the submission ports(465, 587), which require authentication. Unless a relay host is configured, outgoing email will leave the server via port 25(thus outbound traffic must not be blocked by your provider or firewall).
```

Comment:
```suggestion
2. Receives email (usually from other SMTP servers) and filters for spam and viruses. For submitting outgoing mail you should prefer the submission ports (465, 587), which require authentication. Unless a relay host is configured, outgoing email will leave the server via port 25 (thus outbound traffic must not be blocked by your provider or firewall).
```

Reply:
> (usually from other SMTP servers)

Is that not covered with the following sentence about submitting outgoing mail? 

Port 25 serves the purpose of an MTA (transfer), but can also handle submissions without authentication, such as from your own containers or as an open relay. The user is encouraged to use the submission ports (MSA), so they should only need to think of port 25 for receiving email from a third-party to an inbox, not to be sent out of the server(submission).

Reply:
Yes, although I cannot find that sentence anywhere... where's that from?

The note you deleted seemed more straightforward: *Note: Port 25 is only for receiving email from other mailservers and not for submitting email. You need to use port 465 or 587 for this.* It clarifies the difference among transfer and submission.

Reply:
> It clarifies the difference among transfer and submission.

Port 25 is for an MTA, an MTA can handle both transfer and submission, however port 25 is unauthenticated, thus exposing it to the public internet for submission(not possible by default this image is configured to restrict submission to local IP iirc?) would allow spammers to use the server as an open relay. You could use it to submit email from your containers without them having to provide authentication to do so.

To reduce risk, it's simpler to direct users ignore port 25 for submission, and just deal with the MSA on port 587 / 465, these require authentication upfront making them safer. Before the MSA was a thing though, it's what the MTA handled as well, as such it is capable of that.

Insisting that port 25 is only for receiving and incapable of submission can be a bit confusing if you've learned about that it can prior be used for such. It would be fine if it was more clear on the fact it was best practice and for security reasons to only use port 25 for receiving email and the submission ports for submitting. 

Not only that, I recall reading here that if you do submit to port 25, it would still go through the spam filters and other configuration that differentiates it from submission ports in this image. I haven't looked into that or verified, but that would make the port less attractive to submit to if your own email being sent out gets blocked from doing so..

> Yes, although I cannot find that sentence anywhere... where's that from?

What sentence? I clearly state after the receiving email sentence, that preference should be given to submission ports when you want to send email yourself(either via some email client  or your own containers).

The other half of `2` isn't covered by the previous note that I've replaced. My note only indicates port 25 as capable of receiving email.